torch/csrc/lazy/core/shape_inference.h

```diff
@@ -64,6 +64,9 @@ TORCH_API std::vector<Shape> compute_shape_sum(const at::Tensor & self, c10::opt
 TORCH_API std::vector<Shape> compute_shape__to_copy(const at::Tensor & self, c10::optional<at::ScalarType> dtype, c10::optional<at::Layout> layout, c10::optional<at::Device> device, c10::optional<bool> pin_memory, bool non_blocking, c10::optional<at::MemoryFormat> memory_format);
 TORCH_API std::vector<Shape> compute_shape_trace(const at::Tensor & self);
 TORCH_API std::vector<Shape> compute_shape_zero_(at::Tensor & self);
+TORCH_API std::vector<Shape> compute_shape_mul(const at::Tensor & self, const at::Tensor & other);
+TORCH_API std::vector<Shape> compute_shape_mul_(at::Tensor & self, const at::Tensor & other);
+TORCH_API std::vector<Shape> compute_shape_mul_out(at::Tensor& out, const at::Tensor& self, const at::Tensor& other);
```

Comment:
Nice catch- looks like removing `mul` also breaks LazyTensor now - I'll mention this in the wiki page (basically, we now lose "meta tensor" support for the `mul` function, so LazyTensor expects it to be manually implemented here instead).